.github/workflows/build_notarized.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         if [[ "${{ env.is-official-release }}" == "true" ]]; then
           echo "upload-to=s3" >> $GITHUB_OUTPUT
           echo "upload-to=s3" >> $GITHUB_ENV
-        elif [[ -n "${{ env.asana-task-url }}" ]]; then
+        else
```

Comment:
Don't require asana-task-url input to be provided and set `upload-to` to `s3testbuilds` for all cases when it's not an official release build.